src/Picqer/Financials/Exact/Persistance/Downloadable.php

```diff
@@ -17,7 +17,11 @@ public function download()
     {
         $client = new Client();
 
-        $uri = $this->Url . '&Download=1';
+        if ($this->Url) {
+            $uri = $this->Url;
```

Comment:
Is this correct, since the old situation had the download query param which is now removed?

Reply:
Oops, I don't think that `&Download=1` should be removed. I will update the PR.